packages/q/qtifw/xmake.lua

```diff
@@ -0,0 +1,58 @@
+package("qtifw")
+    set_kind("binary")
+    set_homepage("https://doc.qt.io/qtinstallerframework/")
+    set_description("The Qt Installer Framework provides a set of tools and utilities to create installers for the supported desktop Qt platforms: Linux, Microsoft Windows, and macOS.")
+    set_license("GPL-3")
+
+    add_deps("aqt")
+    if is_host("linux") then
+        add_deps("libxcb")
+    end
+
+    on_install("@linux", "@windows", "@macosx", function (package)
```

Comment:
Could I make package to be installed on msys platform here?

Reply:
you can try .